src/Adapter/GuzzleHttpAdapter.php

```diff
@@ -1,19 +1,38 @@
 <?php
 
+/*
+ * (c) Henrik Bjornskov <henrik@bjrnskov.dk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 namespace Stampie\Adapter;
 
 use GuzzleHttp\ClientInterface;
 use Stampie\Attachment;
 
+/**
+ * @package Stampie
+ */
 class GuzzleHttpAdapter implements Stampie\Adapter
 {
+    /**
+     * @var ClientInterface
+     */
     private $client;
 
+    /**
+     * @param ClientInterface $client
```

Comment:
this phpdoc does not provide any value. The signature already gives all this info